### DISCUSSION TITLE
Company-mode Organization Profile Picture

### DISCUSSION THREAD

**Discussion by yugaego** [company-mode/company-mode#1371] [General]
I suggest to add an icon to the company-mode organization profile.
It can be improved or replaced if someone with better graphical skills & ideas comes around.

![company-mode-5](https://user-images.githubusercontent.com/296460/217519782-84b9e80b-5cf2-4cf7-a042-39fead6d13aa.png)

**Comment by dgutov** [comment]
Hey, that's pretty good. And explanatory as well.

I've set it as the organization icon, thank you.

It should work at least until we come up with a better name for this package (something that's been an issue for a bunch of packages I maintain).
